windows/runner.ps1

```diff
@@ -101,9 +95,3 @@ Write-Output "Wrote to $outfile"
 # Cleanup:
 $jobs | Remove-Job
 
-# remove BOM from JSON file (the 90's called, they want their file encoding back!)
-.\dos2unix.exe -r $outfile 2>&1
-
-# Sync with Tidal:
-tidal sync servers $outfile 2>&1 
```

Comment:
Do we really need to remove these lines as well?

Reply:
No we don't _really_ need to. But, if we don't remove this, then the README.md needs to be updated to show how to properly invoke and use this. I'm in favour of keeping this line, its simpler for users, and updating the readme.